### PR TITLE
DEVELOPER-1435 JBDS-3562 PR to move to the new jboss-devstudio-9.0.0.GA-CVE-2015-7501-installer-eap.jar, and update the name label on it. 

### PR DESCRIPTION
**Pull request by nickboldt**
DO NOT APPLY until the jar's in GG / CSP. Also... do we want a separate release entry, or just a change to the existing 9.0.0.GA entry? Should we update the release date from Oct 2 to Dec 8?

### CODE REVIEW

**Comment by alexeykazakov** [comment]
I don't think we should change the release date since we can't have a seporate date for the eap installer (can we?)

**Comment by nickboldt** [comment]
nope, not without some more extensive changes to the way the yaml works.

File: _config/products.yml [hunk 1]
```diff
@@ -87,8 +87,8 @@ devstudio:
           - name: Stand-alone Installer
             url: https://www.jboss.org/download-manager/content/origin/files/sha256/b5/b55d3b959d4922f69338095aff4d3cea639766a7c07850e539ebc0b9029e8902/jboss-devstudio-9.0.0.GA-installer-standalone.jar
             file_size: 488MB
-          - name: Installer With EAP 6.4.0
-            url: https://www.jboss.org/download-manager/content/origin/files/sha256/4f/4fe77f731aa2a46a4d8568786c0c7c2d486e0926025d06a95360e9e767a2af94/jboss-devstudio-9.0.0.GA-installer-eap.jar
+          - name: Installer With EAP 6.4.0 CVE-2015-7501 / BZ 1281963
+            url: https://www.jboss.org/download-manager/content/origin/files/sha256/1e/1efef4d2b109efd9e8136d02067a8d62e00621e5b2b1530811bf52b5aa2fca6f/jboss-devstudio-9.0.0.GA-CVE-2015-7501-installer-eap.jar
```
Comment (maxandersen) [inline _config/products.yml]: BZ-1281963 is not publically accessible thus not relevant/good to put into this name.

**Comment by maxandersen** [comment]
about date, could add that to the name (15-Dec-2015)

**Comment by alexeykazakov** [comment]
Looks good.

**Comment by alexeykazakov** [comment]
Pushed to master and production.

**Comment by maxandersen** [comment]
Thanks - I pushed a small update to fix typo and reduce the title length so it does not span multiple lines.

**Pull request merged**